src/ui/src/predefinedfilterscombobox.cpp

```diff
@@ -172,6 +177,7 @@ void PredefinedFiltersComboBox::setTitle( const QString& title )
 {
     auto* titleItem = new QStandardItem( title );
     model_->insertRow( 0, titleItem );
+    delete titleItem;
```

Comment:
As far as I understand after insertRow call the ownership should be transferred to the model.

Reply:
> As far as I understand after insertRow call the ownership should be transferred to the model.

leave this change?